platforms/web/src/index.tsx

```diff
@@ -19,8 +21,12 @@ configureEnv({
   APP_PLAYER_LICENSE_KEY: import.meta.env.APP_PLAYER_LICENSE_KEY,
 
   APP_FOOTER_TEXT: import.meta.env.APP_FOOTER_TEXT,
+  APP_BODY_FONT: import.meta.env.APP_BODY_FONT,
+  APP_BODY_ALT_FONT: import.meta.env.APP_BODY_ALT_FONT,
```

Comment:
Hm, maybe we should change the approach here in the future. Some of the envs relate to styling some of the, to the core functionality. Maybe we need an additional local config somewhere, maybe something different. Just thinking out loud.